Appveyor install libevent[thread] vcpkg

sipsorcery opened the pull request:
As per #17586 the default libevent vcpkg install now has thread disabled. This PR installs libevent with the thread feature enabled.

laanwj commented:
ACK (assuming this passes AppVeyor)

Please shorten the commit message though—"As per #17586 the default libevent vcpkg install now has thread disabled. This PR installs libevent with the thread feature enabled." would be better as a commit body than title.

sipsorcery committed:
Appveyor install libevent[thread] vcpkg

As per #17586 the default libevent vcpkg install now has thread disabled. This PR installs libevent with the thread feature enabled.
